CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.8.0 (December 21, 2022)
```

Comment:
Non blocker: why minor update for this instead of patch?

Reply:
Good question. We should use patch instead for this. Next time.